src/components/books.js

```diff
@@ -1,25 +1,26 @@
-import React, { useState } from 'react';
-import Book from './book';
-import AddBook from './AddBook';
+/* eslint-disable react/prop-types */
+/* eslint-disable import/extensions */
+import React from 'react';
+import Book from './Book';
+import AddBook from './addBook';
```

Comment:
- [ ] As pointed out by the previous reviewer, kindly enable linters here as they help us write standardized code. Here are steps on how to resolve this:

  - Install the `'prop-types'` package with this command `npm i -S prop-types`
  - At the top this class, here, import the PropTypes, like so: `import PropTypes from 'prop-types';`
  - At the end of this file, change this code:
    
    ```
    Books.defaultProps = {
      books: [],
    };
    ```
    to this:
    ```
    Books.propTypes = {
      bookList: PropTypes.instanceOf(Array).isRequired,
    };
    ```
    You can read more about type checking with prop type [here](https://reactjs.org/docs/typechecking-with-proptypes.html)

- [ ] As highlighted by the previous reviewer you still need to fix this import as your app does not run without them being fixed. You can change to this:

  ```
  import Book from './book';
  import AddBook from './AddBook';
  ```